src/es/components/controllers/autoCompleteLocation/AutoCompleteLocation.js

```diff
@@ -43,14 +43,11 @@ export default class AutoCompleteLocation extends Shadow() {
     this.RESOLVE_MSG = 'LOADED'
   }
 
-  connectedCallback() {
-    Promise
-      .resolve(this.loadDependency())
-      .then(() => {
-        this.addEventListener(this.getAttribute('request-auto-complete') || 'request-auto-complete-location', this.requestAutoCompleteListener)
-        this.addEventListener(this.getAttribute('auto-complete-selection') || 'auto-complete-location-selection', this.clickOnPredictionListener)
-      })
+  async connectedCallback() {
+    this.addEventListener(this.getAttribute('request-auto-complete') || 'request-auto-complete-location', this.requestAutoCompleteListener)
+    this.addEventListener(this.getAttribute('auto-complete-selection') || 'auto-complete-location-selection', this.clickOnPredictionListener)
     this.addEventListener("client-location-coords", this.clickOnLocateMe)
+    await this.loadDependency()
```

Comment:
I think this has no impact. I originally thought you may wait within the requestAutoCompleteListener  and clickOnPredictionListener for the loadDependency.